### ISSUE TITLE
Can we use exception filters again?

### ISSUE THREAD

**Issue by SimonCropp** [getsentry/sentry-dotnet#1626]
[here](https://github.com/getsentry/sentry-dotnet/blob/0db8399b98c8730489c3555ce85481051b74fa20/src/Sentry/Internal/Http/CachingTransport.cs#L110) 

```
// Can't use exception filters because of a Unity 2019.4.35f IL2CPP bug
// https://github.com/getsentry/sentry-unity/issues/550
if (ex is OperationCanceledException && _workerCts.IsCancellationRequested)
{
```

But it seems [that issue](https://github.com/getsentry/sentry-unity/issues/550) is resolved?

**Comment by SimonCropp** [comment]
ping @bitsandfoxes

**Comment by SimonCropp** [comment]
give the underlying issue is resolved in  2019.4.38f1 https://issuetracker.unity3d.com/issues/il2cpp-build-fails-on-try-block-ends-without-any-catch-finally-nor-fault-handler
i will do a PR to put the exception filter back
